src/frontends/onnx/frontend/src/version_range.hpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+namespace ngraph {
```

Comment:
Can this namespace be `ov` as this is new file?

Reply:
Can we change namespace in whole frontend? :)

Reply:
changed locally, for a global change we have separate ticket - 102019 ;)